File: Python-Programing-Basics/While Loop/While Loop - Lab/05_account_balance/account_balance.py
total = 0
command = input()

while command != "NoMoreMoney":
    command = float(command)
    if command >= 0:
        total += command
        print(f"Increase: {command:.2f}")
    else:
        print("Invalid operation!")
        break
    command = input()

print(f"Total: {total:.2f}")

File: JavaScript/Programming Basics - JavaScript/While Loop/accountBalance.js
function updateBalance(input){
    let balance = 0;
    let i = 0;
    let action = input[i];
    while(action !== 'NoMoreMoney'){
        action = Number(action);
        if (action < 0){
            console.log('Invalid operation!');
            break;
        }
        balance += action;
        console.log(`Increase: ${action.toFixed(2)}`);
        i++;
        action = input[i];
    }
    console.log(`Total: ${balance.toFixed(2)}`);
}

updateBalance(["120",
"45.55",
"-150"]);